source/gui/settingsDialogs.py

```diff
@@ -504,7 +502,7 @@ class MultiCategorySettingsDialog(SettingsDialog):
 	"""
 
 	title = ""
-	categoryClasses: typing.List[typing.Type[SettingsPanel]] = []
+	categoryClasses: list[type[SettingsPanel]] = []
```

Comment:
Here and everywhere else, type hints have been modified / modernized, even on parts of the code that are unrelated to this PR.

For trackability, I'd suggest to modernize type hints on change only code lines that are related to this PR, so that we do not get this PR mentioned on unrelated code lines with `git blame`.

Note that, if I remember correctly, an issue is open for a general modernization of type hints. Such changes should rather be implemented in a dedicated PR so that its commit can be added to `.git-blame-ignore-revs`, so that it'll be ignored by `git blame`.